src/zenml/services/service.py

```diff
@@ -232,21 +290,28 @@ def update_status(self) -> None:
             "Running status check for service '%s' ...",
             self,
         )
-        state, err = self.check_status()
-        logger.debug(
-            "Status check results for service '%s': %s [%s]",
-            self,
-            state.name,
-            err,
-        )
-        self.status.update_state(state, err)
+        try:
```

Comment:
Just a very minor thing. While reading the docstring of this method, I realized that it says "Update the service of the service". We can also fix that here 😅